extensions/ce/user-access-resources.adoc

```diff
@@ -0,0 +1,33 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="user-access-resources"]
+= User access to extension resources
+include::_attributes/common-attributes.adoc[]
+:context: user-access-resources
+
+toc::[]
+
+After a cluster extension has been installed and is being managed by {olmv1-first}, the extension can often provide `CustomResourceDefinition` objects (CRDs) that expose new API resources on the cluster. Cluster administrators typically have full management access to these resources by default, whereas non-cluster administrator users, or _regular users_, might lack sufficient permissions.
+
+{olmv1} does not automatically configure or manage role-based access control (RBAC) for regular users to interact with the APIs provided by installed extensions. Cluster administrators must define the required RBAC policy to create, view, or edit these custom resources (CRs) for such users.
+
+[NOTE]
+====
+The RBAC permissions described for user access to extension resources are different from the permissions that must be added to a service account to enable {olmv1}'s initial installation of a cluster extension itself. For more on RBAC requirements while installing an extension, see "Cluster extension permissions" in "Managing extensions".
+====
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../../extensions/ce/managing-ce.adoc#managing-ce["Managing extensions" -> "Cluster extension permissions"]
```

Comment:
The "Cluster extension permissions" section is not actually live yet. Will update the anchor when that PR merged.